contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
Right now for rpm and deb we're going off what meson.build will spit out (rpm does it through ninja-dist and deb does it by parsing meson.build).  When it comes time to cut a release that release should represent the accurate version of what it is.